mypy/config_parser.py

```diff
@@ -78,6 +78,7 @@ def split_and_match_files(paths: str) -> List[str]:
     'junit_xml': expand_path,
     # These two are for backwards compatibility
     'silent_imports': bool,
+    'no_site_packages': bool,
```

Comment:
I think you stuck this between two configs the comment above points to, so you should probably put it below `almost_silent`.

Reply:
Thanks for responding so fast. I have applied the suggestion. Is it possible to add a test somewhere? I performed 2 manual test-cases (both testing combination of CLI and config version of the flag).

Reply:
Good question! The best place for a test would probably be in https://github.com/python/mypy/blob/0e8e135548a4d2fde42f93e2a2a56802af019263/test-data/unit/cmdline.test.

Reply:
I have an idea that the following is not monkey-proof:
```
[case testNoSitePackagesFlag]
# cmd: mypy emtpy_test.py --no-site-packages --verbose
[file emtpy_test.py]
pass
[out]
LOG:  Configured Executable:  None
```

And the test with pytest also fails, as it is not the right line, so this is just to indicate what I'd like to achieve. I'd like the test the fact that the options class ends up with the right value, so this is one of the way I thought it might be achievable.
Any thoughts on how to make this more robust would be appreciated as my knowledge of the test syntax is limited.

Reply:
Ah, the best way to test this may be to add to https://github.com/python/mypy/blob/master/mypy/test/testpep561.py

PEP 561 tests are somewhat special.

Reply:
I find line [Line 235](https://github.com/python/mypy/blob/1677efdfd7c37ee91d64bc018b12d6bfda042655/mypy/test/testpep561.py#L235) on building a config very handy. That's definitely the way to go.

But [Line 249](https://github.com/python/mypy/blob/1677efdfd7c37ee91d64bc018b12d6bfda042655/mypy/test/testpep561.py#L249) about accessing the mypy module(s), makes me worried that I still have to scan `stdout` (see [main.py line 63](https://github.com/python/mypy/blob/1677efdfd7c37ee91d64bc018b12d6bfda042655/mypy/main.py#L63)). Is that acceptible in your view? It is prone to false-positives.

Reply:
Yes, you will have to match against stdout, which isn't ideal, but how most of our testing works. You can see how that is done for other tests based on this: https://github.com/python/mypy/blob/master/mypy/test/testpep561.py#L55. If the test is static there should be no false positive.

Reply:
I have added a static test like previously discussed by extending the ExampleProgram with args. Any idea why it's failing on importing `typedpkg`? I've copied the other tests, so this is unexpected.